CONTRIBUTING.md

```diff
@@ -156,6 +152,13 @@ How to enable popular IDEs to run the unittests:
 * PyCharm: PyCharm supports test discovery and execution for the Python unittest framework. You may need to configure
 PyCharm to use the unittest framework. Click PyCharm > Preferences > Tools > Python Integrated Tools and under the 
 Testing heading, choose `Unittests` from the `Default test runner` dropdown.
+* VSCode: VSCode supports test discovery and execution for the Pytyhon unittest framework. Under the *Testing* panel, configure the tests to use the unittest framework and search for tests beginning with *test_**. 
+  
+A *.env* file will also need to be configured with the following variables set:
+* GOOGLE_APPLICATION_CREDENTIALS - The path to your Google Cloud Project credentials
+* TEST_GCP_PROJECT_ID - Your GCP Project ID
+* TEST_GCP_DATA_LOCATION - The location of your GCP Project
+* TEST_GCP_BUCKET_NAME - The name of your GCP testing bucket 
```

Comment:
Some unit tests for OpenAlex requires AWS credentials to create buckets and transfer to GCS. You may need to briefly mention this or just link to a docs page.

Reply:
It's only relevant when changing those tests, but yeah I'll make mention of it